packages/design-system-react/src/components/AvatarBase/AvatarBase.tsx

```diff
@@ -33,7 +33,7 @@ export const AvatarBase = forwardRef<HTMLDivElement, AvatarBaseProps>(
 
     const mergedClassName = twMerge(
       // Base styles
-      'inline-flex flex-shrink-0 items-center justify-center overflow-hidden bg-section',
+      'inline-flex shrink-0 items-center justify-center overflow-hidden bg-section',
```

Comment:
Both classnames work but this aligns more with [tailwind docs](https://tailwindcss.com/docs/flex-shrink)